src/components/Footer.jsx

```diff
@@ -44,11 +43,4 @@ const Footer = ({
   </div>
 )
 
-Footer.propTypes = {
-  isDeleteDisabled: PropTypes.bool,
-  isSaveDisabled: PropTypes.bool,
-  deleteCallback: PropTypes.func,
-  saveCallback: PropTypes.func,
-}
-
```

Comment:
Could I check why the proptypes here are removed?

Reply:
the proptypes are never used as these don't exist as props @__@